service/src/main/java/com/ge/predix/acs/service/policy/evaluation/ResourceAttributeResolver.java

```diff
@@ -44,9 +42,10 @@
      * @param requestResourceUri
      *            URI of the resource from the policy evaluation request
      */
-    public ResourceAttributeResolver(final PrivilegeManagementService privilegeService, final String requestResourceUri,
-            final Set<Attribute> supplementalResourceAttributes) {
-        this.privilegeService = privilegeService;
+    public ResourceAttributeResolver(final AttributeStore resourceAttributeStore,
+                                     final String requestResourceUri,
+                                     final Set<Attribute> supplementalResourceAttributes) {
```

Comment:
this format is better, though is not per our current project settings. pls re-concile